docs/contribute/release_checklist.md

```diff
@@ -6,6 +6,7 @@ permalink: /contributing/release-checklist
 
 ## Before the release
 
+- Make sure version number is correct or update version in Config.kt 
```

Comment:
The "After the release" item still mentions updating the version in build.gradle btw. :)